test/term-definition-length.test.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import path from 'path';
+
+const __filename = path.resolve();
+
+const web3TermsPath = path.join(__filename, '/data/terms.json');
+
+describe('Check definition Length', () => {
+  // Read the file
+  const data = fs.readFileSync(web3TermsPath, 'utf8');
+
+  if (!data) {
+    return [];
+  }
+  const parsedData = JSON.parse(data);
+
+  parsedData.forEach(termObject => {
+    it(`"${termObject.term}" definition should be less than or equal 290, got ${termObject.meaning?.length}`, () => {
+      expect(termObject.meaning?.length).toBeLessThanOrEqual(290);
```

Comment:
I think that the assertion is wrong because as mentioned in [41](https://github.com/Olanetsoft/web3terms/issues/41) the length should be maximum 280 characters long while you are asserting that it is 290 or less. If @Olanetsoft could confirm this it would be great.

Reply:
Thanks for mentioning that. Will fix that alongside